src/fluxo.object_store.js

```diff
@@ -13,23 +14,19 @@ class ObjectStore {
 
     this.data = {};
 
-    this.storeAttributesEventsCanceler = {};
+    this.radio = new Radio;
 
     this.computed = (this.constructor.computed || {});
 
-    this.signedEventsCancelers = [];
-
-    this.events = {};
+    this.storeAttributesEventsCanceler = {};
 
     this.setDefaults();
 
     this.set(data);
 
     this.registerComputed();
 
-    this.on(["change"], function () {
-      delete this.lastGeneratedJSON;
-    });
+    this.radio.on(["change"], () => delete this.lastGeneratedJSON);
```

Comment:
@samuelsimoes the events must be binded to the store?

I believe it's better to not do that since the store is already passed as an argument to the callback (in that case I'm using an arrow function but we could use the received store).

What you think?